cpp/src/plasma/client.cc

```diff
@@ -501,9 +501,19 @@ Status PlasmaClient::Abort(const ObjectID& object_id) {
 }
 
 Status PlasmaClient::Delete(const ObjectID& object_id) {
```

Comment:
In `client.h` we should probably clarify the behavior of this method. In particular, if a client calls `Delete`, that does not guarantee that the object will be deleted. In particular, if the object is in use by another client, then the `Delete` call is a no-op, right? Can you mention this in the documentation for this method in `client.h`?

Reply:
No, it doesn't guarantee the object will be deleted for some reasons.